src/user/login/login.html

```diff
@@ -23,7 +23,7 @@
           <label>Log In</label>
         </div>
 
-        <form name="formUser" ng-submit="$ctrl.login($event)" layout="column">
+        <form name="formUser" ng-submit="$ctrl.login($event)" layout="column" ng-if="$ctrl.localLoginEnabled">
```

Comment:
What is the reason to show the login view if `localLogin` is disabled ?
IMHO, login view is more-or-less useless if we are doing only oauth authentication.

WDYT ?

Reply:
(see above reply) this is to handle the case where there are multiple external idp. let's say google and a custom oauth. having all of them in the top bar will not look too good, so we display the login form with the list of the providers.

please let me know if you prefer a different logic.